recipes/cython-blis/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cython-blis" %}
+{% set version = "0.2.4" %}
+{% set sha256sum = "2d4ca1508fd6229c7994fc17ba324083a5b83f66612c8ea62623a41a1768b030" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/blis/blis-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  skip: true  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - clangdev  # [win or osx]
```

Comment:
This doesn't seem right. Doesn't compiler('c') on osx resolve to clang?

Reply:
I'm not sure. I'll remove it and check.